### ISSUE TITLE
We need a demo scene or something

### ISSUE THREAD

**Issue by strangeways-dev** [dreaw131313/MotionMatchingByDreaw#5]
There's no way to know how to implement this.
I know documentation can take a while but a sample scene we can learn from would be a great help.

Would love to use and contribute to this but don't even know where to start.

**Comment by VicPrzz00** [comment]
I agree, a demo scene is needed to help people see how it's set up, and save them from having to buy the expensive asset store version
